tests/test_code_format.py

```diff
@@ -5,27 +5,29 @@
 
 
 class TestCodeFormat(TestCase):
-    """ This test class should ensure all earthdata-varinfo Python code adheres
-        to standard Python code styling.
-
-        Ignored errors and warnings:
-
-        * E501: Line length, which defaults to 80 characters. This is a
-                preferred feature of the code, but not always easily achieved.
-        * W503: Break before binary operator. Have to ignore one of W503 or
-                W504 to allow for breaking of some long lines. PEP8 suggests
-                breaking the line before a binary operator is more "Pythonic".
-
+    """This test class should ensure all earthdata-varinfo Python code adheres
+    to standard Python code styling.
+
+    Ignored errors and warnings:
+
+    * E501: Line length, which defaults to 80 characters. This is a
+            preferred feature of the code, but not always easily achieved.
+    * W503: Break before binary operator. Have to ignore one of W503 or
+            W504 to allow for breaking of some long lines. PEP8 suggests
+            breaking the line before a binary operator is more "Pythonic".
+    * E203, E701: This repository uses black code formatting, which deviates
+                  from PEP8 for these errors.
     """
+
     @classmethod
     def setUpClass(cls):
         cls.python_files = Path('varinfo').rglob('*.py')
 
     def test_pycodestyle_adherence(self):
-        """ Ensure all code in the `varinfo` directory adheres to PEP8 defined
-            standards.
+        """Ensure all code in the `varinfo` directory adheres to PEP8 defined
+        standards.
 
         """
-        style_guide = StyleGuide(ignore=['E501', 'W503'])
+        style_guide = StyleGuide(ignore=['E501', 'W503', 'E203', 'E701'])
```

Comment:
[See here for reference](https://black.readthedocs.io/en/stable/guides/using_black_with_other_tools.html#pycodestyle)